libsolidity/ast/Types.h

```diff
@@ -51,6 +51,15 @@ using FunctionTypePointer = std::shared_ptr<FunctionType const>;
 using TypePointers = std::vector<TypePointer>;
 using rational = boost::rational<dev::bigint>;
 
+inline rational makeRational(bigint const& _numerator, bigint const& _denominator)
```

Comment:
I think this would be better suited in an anonymous namespace inside the cpp file or in a more central place inside libdevcore.

Reply:
Actually, I don't think so - in this header we introduce the ``rational`` alias and - if we want to stay compatible with the buggy boost version - we will always want to use ``makeRational``, when we create a rational from a numerator and denominator, so I think it makes a lot of sense to have it right here below the ``using rational = ...``...

Reply:
Or if we want to move it, I'd at least move the ``using rational = ...`` part as well.

Reply:
Cleanly moving ``using rational = ...`` to ``libdevcore`` would require changing it's namespace, though - which we could do, but which may have side-effects. So actually I think right here is not too bad of a place for this for now :-).